packages/opencensus-exporter-zpages/src/zpages-frontend/page-handlers/rpcz.page-handler.ts

```diff
@@ -0,0 +1,237 @@
+  CLIENT_RECEIVED_BYTES_PER_RPC = 'grpc.io/client/received_bytes_per_rpc',
+  CLIENT_ROUDTRIP_LATENCY = 'grpc.io/client/roundtrip_latency',
+  CLIENT_SERVER_LATENCY = 'grpc.io/client/server_latency',
+  CLIENT_STARTED_RPCS = 'grpc.io/client/started_rpcs',
+  SERVER_RECEIVED_MESSAGES_PER_RPC = 'grpc.io/server/received_messages_per_rpc',
+  SERVER_RECEIVED_BYTES_PER_RPC = 'grpc.io/server/received_bytes_per_rpc',
+  SERVER_SENT_MESSAGES_PER_RPC = 'grpc.io/server/sent_messages_per_rpc',
+  SERVER_SENT_BYTES_PER_RPC = 'grpc.io/server/sent_bytes_per_rpc',
+  SERVER_SERVER_LATENCY = 'grpc.io/server/server_latency',
+  SERVER_STARTED_RPCS = 'grpc.io/server/started_rpcs'
+}
+
+enum DefaultViews {
+  CLIENT_SENT_BYTES_PER_RPC = 'grpc.io/client/sent_bytes_per_rpc',
+  CLIENT_RECEIVED_BYTES_PER_RPC = 'grpc.io/client/received_bytes_per_rpc',
+  CLIENT_ROUDTRIP_LATENCY = 'grpc.io/client/roundtrip_latency',
+  CLIENT_COMPLETED_RPCS = 'grpc.io/client/completed_rpcs',
+  CLIENT_STARTED_RPCS = 'grpc.io/client/started_rpcs',
+  SERVER_RECEIVED_BYTES_PER_RPC = 'grpc.io/server/received_bytes_per_rpc',
+  SERVER_SENT_BYTES_PER_RPC = 'grpc.io/server/sent_bytes_per_rpc',
+  SERVER_SERVER_LATENCY = 'grpc.io/server/server_latency',
+  SERVER_COMPLETED_RPCS = 'grpc.io/server/completed_rpcs',
+  SERVER_STARTED_RPCS = 'grpc.io/server/started_rpcs'
+}
+
+export class RpczPageHandler {
+  constructor(private statsParams: StatsParams) {}
+
+  /**
+   * Generate Zpages RPC HTML Page
+   * @param json If true, JSON will be emitted instead. Used for testing only.
+   * @returns output HTML
+   */
+  emitHtml(json: boolean): string {
+    /** template HTML */
+    const rpczFile =
+        ejs.fileLoader(`${templatesDir}/rpcz.ejs`, 'utf8').toString();
+    /** CSS styles file */
+    const stylesFile =
+        ejs.fileLoader(`${templatesDir}/styles.min.css`).toString();
+    /** EJS render options */
+    const options = {delimiter: '?'};
+
+    const rpcViews = this.statsParams.registeredViews.filter(
+        view => view.name.indexOf('http') < 0);
+    const rpczData: RpczData = {measuresSent: {}, measuresReceived: {}};
+
+    for (const view of rpcViews) {
+      const recordedData = this.statsParams.recordedData[view.name];
+      for (const snapshot of recordedData) {
+        let method = snapshot.tags['grpc_client_method'];
+        let zMeasures = rpczData.measuresSent;
+
+        // Switches to received data if it's a server
+        if (!method) {
+          method = snapshot.tags['grpc_server_method'];
+          zMeasures = rpczData.measuresReceived;
+        }
+
+        if (method) {
+          if (!zMeasures[method]) {
+            zMeasures[method] = this.newEmptyZMeasure();
+          }
+
+          // Fills the output columns for that method
+          if (view.name === DefaultViews.CLIENT_SENT_BYTES_PER_RPC ||
+              view.name === DefaultViews.SERVER_SENT_BYTES_PER_RPC) {
+            const distribution = snapshot as DistributionData;
```

Comment:
Can you throw an error if it's not a `DistributionData`, and remove this cast? (Also in below)